frontend/src/scenes/insights/filters/BreakdownFilter/BreakdownTagMenu.tsx

```diff
@@ -85,6 +91,58 @@ export const BreakdownTagMenu = (): JSX.Element => {
                         Do not bin numeric values
                     </LemonButton>
                 </>
+            ) : (
+                <>
+                    <LemonSwitch
+                        fullWidth
+                        className="min-h-10 px-2"
+                        checked={!breakdown?.breakdown_hide_other_aggregation}
+                        onChange={() =>
+                            updateBreakdown({
+                                ...breakdown,
+                                breakdown_hide_other_aggregation: !breakdown?.breakdown_hide_other_aggregation,
+                            })
+                        }
+                        label={
+                            <div className="flex gap-1">
+                                <span>Group remaining values under "Other"</span>
+                                <Tooltip
+                                    title={
+                                        <>
+                                            If you have over {breakdown?.breakdown_limit ?? 25} breakdown options, the
+                                            smallest ones are aggregated under the label "Other". Use this toggle to
+                                            show/hide the "Other" option.
+                                        </>
+                                    }
+                                >
+                                    <IconInfo className="text-muted text-xl shrink-0" />
+                                </Tooltip>
+                            </div>
+                        }
+                    />
+                    <div>
+                        <LemonButton
+                            onClick={() => {
+                                updateBreakdown({ breakdown_limit: breakdownLimit })
+                            }}
+                            status="stealth"
+                            active={histogramBinsUsed}
+                            fullWidth
+                        >
+                            Breakdown limit:{' '}
+                            <LemonInput
+                                min={1}
+                                value={breakdownLimit}
+                                onChange={(newValue) => {
+                                    setBreakdownLimit(newValue ?? 25)
```

Comment:
What doesn't work here for me is typing a value and then focusing somewhere else, thereby closing the menu. That doesn't seem to trigger onChange.

Reply:
Indeed, the UX here is poor. I'll improve it in the next PR as it seems I made the breakdown pills huge now 
<img width="292" alt="image" src="https://github.com/PostHog/posthog/assets/53387/a2b60f69-173b-4598-b954-296e85aab567">